### PR TITLE
docs: Add Web Clipper Options

### LINKED ISSUE #44: Idea: web clipper
Evernote, Notion, TiddlyWiki, Joplin all have web clippers. Foam would be even more awesome if it had one.

### PR DESCRIPTION
**Pull request by SanketDG**
Closes #44

### CODE REVIEW

File: docs/meeting-notes/foam-core-2020-07-11.md [hunk 1]
```diff
@@ -123,5 +123,5 @@ How others solve this:
 
 [//begin]: # "Autogenerated link references for markdown compatibility"
 [todo]: ../todo "Todo"
-[Index]: ../index "Foam"
+[Index]: ../../node_modules/socks/docs/examples/index "socks examples"
```
Comment (riccardoferretti) [inline docs/meeting-notes/foam-core-2020-07-11.md:126]: mmm... this looks like a bug in the link generation?

**Review by riccardoferretti — CHANGES_REQUESTED** [review]

**Review by jevakallio — CHANGES_REQUESTED** [review]
@SanketDG thank you for this proposal! We require a recipe to be a bit more than just a link to an extension -- it should explain what the extension does, and how to use it.

Here's an example for a Recipe structure:
https://foambubble.github.io/foam/how-to-write-recipes

**Comment by riccardoferretti** [comment]
@SanketDG just checking whether you have the bandwidth to follow up on this, otherwise we can close the PR for the time being.. thanks!

**Comment by SanketDG** [comment]
Oops this somehow slipped in my notifications, expect an update soon.

**Review by riccardoferretti — APPROVED** [review]

**Comment by riccardoferretti** [comment]
@SanketDG the update of the documentation from #283 created a conflict - can you please regenerate the backlinks and push an updated PR? I will merge it once that's done

File: docs/link-reference-definitions.md [hunk 2]
```diff
@@ -68,6 +68,7 @@ See [[link-reference-definition-improvements]] for further discussion on current
 
 
 [//begin]: # "Autogenerated link references for markdown compatibility"
-[workspace-janitor]: workspace-janitor.md "Janitor"
-[link-reference-definition-improvements]: link-reference-definition-improvements.md "Link Reference Definition Improvements"
+[workspace-janitor]: workspace-janitor "Janitor"
```
Comment (riccardoferretti) [inline docs/link-reference-definitions.md:71]: these links need to include the `.md` extension, you need to change the setting for the wikilink definitions to `withExtension`

File: docs/meeting-notes/foam-core-2020-07-11.md [hunk 3]
```diff
@@ -122,6 +122,6 @@ How others solve this:
 - Unique ids -- could support optionally as part of file name or front matter metadata. Should not be required.
 
 [//begin]: # "Autogenerated link references for markdown compatibility"
-[todo]: ../todo.md "Todo"
-[Index]: ../index.md "Foam"
+[todo]: ../todo "Todo"
+[Index]: ../../node_modules/socks/docs/examples/index "socks examples"
```
Comment (riccardoferretti) [inline docs/meeting-notes/foam-core-2020-07-11.md:126]: interesting... I wonder where this is coming from - feels like a bug in the definition generation..

File: docs/predefined-user-snippets.md [hunk 4]
```diff
@@ -48,16 +48,6 @@ We could provide snippets that expand out into the associated markdown syntax, l
...
-[//end]: # 'Autogenerated link references'
-[//begin]: # 'Autogenerated link references for markdown compatibility'
-[custom-snippets]: custom-snippets 'Adding Custom Snippets'
-[//end]: # 'Autogenerated link references'
-
 [//begin]: # "Autogenerated link references for markdown compatibility"
-[custom-snippets]: custom-snippets.md "Adding Custom Snippets"
-[//end]: # "Autogenerated link references"
\ No newline at end of file
+[custom-snippets]: custom-snippets "Adding Custom Snippets"
```
Comment (riccardoferretti) [inline docs/predefined-user-snippets.md:52]: same as above, link is missing `.md`

File: docs/recipes.md [hunk 5]
```diff
@@ -93,33 +94,34 @@ Thought of a recipe but don't see a category for them? Add them here and we'll o
...
-[github-pages]: github-pages.md "Github Pages"
-[gitlab-pages]: gitlab-pages.md "GitLab Pages"
-[eleventy-and-netlify]: eleventy-and-netlify.md "Eleventy and Netlify"
-[azure-devops-wiki]: azure-devops-wiki.md "Azure DevOps Wiki"
-[vercel]: vercel.md "Vercel"
-[customising-styles]: customising-styles.md "Customising Styles"
-[math-support]: math-support.md "Math Support"
-[katex-math-rendering]: katex-math-rendering.md "Katex Math Rendering"
-[capture-notes-with-drafts-pro]: capture-notes-with-drafts-pro.md "Capture Notes With Drafts Pro"
+[contribution-guide]: contribution-guide "Contribution Guide"
```
Comment (riccardoferretti) [inline docs/recipes.md:97]: you guessed it, links are missing `.md` extension

**Review by riccardoferretti — CHANGES_REQUESTED** [review]

File: docs/meeting-notes/foam-core-2020-07-11.md (hunk 3 above)
Comment (SanketDG) [reply docs/meeting-notes/foam-core-2020-07-11.md:126]: Yeah, I will leave this alone for now.

File: docs/link-reference-definitions.md (hunk 2 above)
Comment (SanketDG) [reply docs/link-reference-definitions.md:71]: How do I make this work? The setting is already there in `docs/.vscode`, but doesn't seem to work.

Comment (riccardoferretti) [reply docs/link-reference-definitions.md]: you need to be sure that:
- you have opened that (and not the parent folder, which has its own `.vscode`, in which case check the setting there
- you don't have a version of the foam extension that is too old (which doesn't support the setting)

let me know if that fixes it

Comment (SanketDG) [reply docs/link-reference-definitions.md]: I see, you have to be in the workspace

**Review by riccardoferretti — APPROVED** [review]
Thanks @SanketDG! Looking good!

**Pull request merged**